src/_base/harness/config/commands.yml

```diff
@@ -308,9 +308,10 @@ command('generate token <length>'):
     }
     echo $string;
 
-command('lighthouse'):
+command('lighthouse [--with-results]'):
```

Comment:
are the results of any use by default?

Reply:
Perhaps, but it's a huge JSON object that could take up a lot of jenkins log disk space if output.

I guess we'd want it stored as a file and as a build artefact for referencing if we want this in the pipeline normally.